src/number.cr

```diff
@@ -166,6 +166,10 @@ struct Number
   # Returns `-1`, `0` or `1` depending on whether `self` is less than *other*, equals *other*
   # or is greater than *other*.
```

Comment:
The doc comment should mention that this method returns `nil` when comparing float with NaN value. Maybe also declare the return type as `Int32?`.

Reply:
Done